lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -71,6 +73,7 @@
                         ['topics/:topic_id/search(/)', _.bind(this.searchTeams, this)],
                         ['topics/:topic_id/create-team(/)', _.bind(this.newTeam, this)],
                         ['topics/:topic_id/:team_id/edit-team(/)', _.bind(this.editTeam, this)],
+                        ['topics/:topic_id/:team_id/edit-team/manage-members(/)', _.bind(this.editTeamMembers, this)],
```

Comment:
Note that edit-team and edit-team/manage-members can be hit directly by any user (though the server does reject the privileged operations). While this is a cool way to verify we handle server errors directly, we may wish to leave these out of the router if the user is not staff/privileged.